app/tsconfig.json

```diff
@@ -10,10 +10,11 @@
         "baseUrl": "."
    },
    "files": [
-	"./src/ts/app.ts",
-    "./src/ts/upload.ts",
-    "./src/ts/play.ts",
-	"./src/ts/video.ts"
+	"src/ts/app.ts",
+    "src/ts/upload.ts",
+    "src/ts/play.ts",
+	"src/ts/video.ts",
+     "src/ts/bootstrap.js"
```

Comment:
js files should not be part of the list of files for the typescript compiler to build. Please remove this line.